crates/sui-bcs-json/examples/main.c

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "sui_bcs_json_bindings.h"
+
+int main (int argc, char const * const argv[])
+{
+
+    // Fail on purpose and retrieve error message
+    char *json_str_0 = "{\"Pure\": [49,50,51] }";
+    const uint8_t *json_result_0;
+    size_t result_len_0;
+    int result_0 = sui_bcs_from_json("Test", json_str_0, &json_result_0 , &result_len_0);
+
+    if (result_0 == 1) {
+        printf("Error code is %d. ", result_0);
+        int buffer = sui_last_error_length();
+        char error_msg [buffer];
```

Comment:
I would personally `malloc` this rather than store it on the stack, given we don't know how big this is going to be (and then `free` it after the `printf`).  Mostly for the sake of setting a good example for people using this as a reference.